samples/product/src/main/conf/synapse/synapse_sample_663.xml

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied. See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+<definitions xmlns="http://ws.apache.org/ns/synapse">
+    <proxy name="SequenceBreakdownSampleProxy" startOnLoad="true" transports="http https">
+        <description/>
+        <target inSequence="StockQuoteSeq"/>
+    </proxy>
+    <sequence name="StockQuoteSeq" onError="StockQuoteErrorSeq">
+        <log level="custom">
+            <property name="Sequence" value="StockQuoteSeq"/>
+            <property name="Description" value="Request recieved"/>
+        </log>
+        <sequence key="CallStockQuoteSeq"/>
+        <sequence key="TransformAndRespondSeq"/>
+    </sequence>
+    <sequence name="StockQuoteErrorSeq">
+        <log level="custom">
+            <property name="Description" value="Error occurred in StockQuoteErrorSeq"/>
+            <property name="Status" value="ERROR"/>
+        </log>
+    </sequence>
+    <sequence name="fault">
+        <!-- Log the message at the full log level with the ERROR_MESSAGE and the ERROR_CODE-->
+        <log level="full">
+            <property name="MESSAGE" value="Executing default 'fault' sequence"/>
+            <property expression="get-property('ERROR_CODE')" name="ERROR_CODE"/>
+            <property expression="get-property('ERROR_MESSAGE')" name="ERROR_MESSAGE"/>
+        </log>
+        <!-- Drops the messages by default if there is a fault -->
+        <drop/>
+    </sequence>
+    <sequence name="main">
```

Comment:
Let's remove main and fault sequences from the sample

Reply:
Removed the main and fault sequences